src/main/java/servlet/post/RegistrationServlet.java

```diff
@@ -0,0 +1,35 @@
+package servlet.post;
+
+import java.io.IOException;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.json.JSONObject;
+
+import com.google.gson.Gson;
+
+import beans.UserBean;
+import helper.HttpServletHelper;
+import model.UserModel;
+
+@WebServlet(name = "RegistrationServlet", urlPatterns = "/register")
+public class RegistrationServlet extends HttpServletHelper {
+	
+	private static final long serialVersionUID = 1L;
+	
+	@Override
+	protected void doPost(HttpServletRequest request, HttpServletResponse response)
+	throws ServletException, IOException {
+		Gson gson = new Gson();
+		UserBean person = null;
+		JSONObject result = new JSONObject();
+		
+		person = gson.fromJson(getRequestBody(request), UserBean.class);		
```

Comment:
just a suggestion, use variable names more similar to its class name, if the class is UserBean i would sugges something like user or bean or userBean which improves readability and keeps to code a little bit cleaner. Also you can just instantiate your user variable immediately  setting it to null is redundant as you will set it to a different value right away.